app/utils/auth.py

```
@@ -6,128 +6,113 @@
+            for field_name, clerk_key in dto_field_to_clerk_key.items()
+            if clerk_key not in payload
+        ]
+
+        return missing_clerk_keys_in_payload, cls(**dto_field_values)
 
 
 def get_current_user(
-		request_from_context: Request,
-		auth_header: HTTPAuthorizationCredentials = Depends(http_bearer_security_schema),
+    request_from_context: Request,
+    auth_header: HTTPAuthorizationCredentials = Depends(http_bearer_security_schema),
```

Ruff (0.11.9)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)